gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/java/io/gravitee/plugin/entrypoint/webhook/WebhookEntrypointConnector.java

```diff
@@ -152,26 +160,24 @@ private Completable sendAndDiscard(
         String requestUri,
         HttpClient httpClient,
         WebhookEntrypointConnectorSubscriptionConfiguration subscriptionConfiguration,
-        Message message
+        Message message,
+        HttpHeaders responseHeaders
     ) {
         // Consume the message in order to send it to the remote webhook and discard it to preserve memory.
         return httpClient
             .rxRequest(HttpMethod.POST, requestUri)
             .flatMap(
                 request -> {
+                    if (responseHeaders != null) {
+                        responseHeaders.forEach(header -> putHeaderIfAbsent(request, header.getKey(), header.getValue()));
+                    }
```

Comment:
imho it should be a put without check, since header from response (so maybe set by a policy at publisher level) are the most important, no ?